source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -308,6 +308,15 @@ void Filter::onComplete(Filters::Common::ExtAuthz::ResponsePtr&& response) {
   Stats::StatName empty_stat_name;
 
   if (!response->dynamic_metadata.fields().empty()) {
+    if (config_->disableDynamicMetadataIngestion()) {
```

Comment:
We probably don't want to fail the whole request just because of this.

We can just ignore the injection but have the logs or even stats for observability purpose. 

cced @ggreenway  as OSS owner of this filter